### PR TITLE
Update somalier

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/nf-core/somalier/relate/nextflow.config [hunk 1]
```diff
@@ -3,7 +3,4 @@ params.prefix = ""
 process {
     publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
 
-    withName:SOMALIER_RELATE {
-        ext.prefix = { params.prefix ? "${params.prefix}." : ""}
```
Comment (asp8200) [inline tests/modules/nf-core/somalier/relate/nextflow.config:7]: I wonder what the purpose of that line was?

Above `params.prefix` was set to `""`, so `ext.prefix` would also always be set to `""`  🤔

Comment (nvnieuwk) [reply tests/modules/nf-core/somalier/relate/nextflow.config:7]: Yeah that's the reason I removed it. `params.prefix` isn't even set...

Comment (asp8200) [reply tests/modules/nf-core/somalier/relate/nextflow.config:7]: Can line 1 also be removed?

Comment (nvnieuwk) [reply tests/modules/nf-core/somalier/relate/nextflow.config:7]: Welp, overlooked that one :D Thanks!

**Review by asp8200 — APPROVED** [review]
Seems fine. Only a minor revision as far as I can tell.

**Review by muffato — APPROVED** [review]

**Comment by ashotmarg** [comment]
Thanks guys for the efficient work updating/improving the code 👍

**Pull request merged**